zgw_consumers/legacy/client.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import warnings
+from typing import IO, Any, Dict, List, Optional, Union
+from urllib.parse import urljoin
+
+from django.utils.module_loading import import_string
+
+import yaml
+import zds_client
+from zds_client import Client
+from zds_client.oas import schema_fetcher
+
+from ..settings import get_setting
+
+logger = logging.getLogger(__name__)
+
+IS_OLD_ZDS_CLIENT = zds_client.__version__ < "2.0.0"
```

Comment:
if we're going towards 1.0, let's drop support for zds client < 2.0 too.

Probably best to create a separate PR for that.

Reply:
Will do in a separate PR